File: Logical/or.py
#저장된 키값
real_egoing = "egoings"  
real_k8805 = "k8805"

#로그인
while True:
    in_key = input("키값을 입력해주세요: ") #문자열
    if real_egoing == in_key or real_k8805 == in_key:
        print("Hello!")
        break;
    print("등록된 정보가 없습니다.")

File: Logical/or.rb
#저장된 키값
real_egoing = "1"
real_k8805 = "ab"

#로그인
print("키값을 입력해주세요: ")
in_str = gets.chomp() #값 입력받기

if real_egoing == in_str or real_k8805 == in_str
    puts("Hello!")  
else
    puts("저장된 정보가 없습니다.")
end